tests/genomics/test_bigscape_loader.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from nplinker.genomics import GCF
+from nplinker.genomics.abc import GCFLoaderBase
+from nplinker.genomics.bigscape import BigscapeGCFLoader
+from .. import DATA_DIR
+
+
+class TestBigscapelGCFLoader:
+
+    @pytest.fixture
```

Comment:
what is the advantage of using this here instead of having a class attribute self.loader? Just for me to understand :)

Reply:
`fixture` is a great feature of pytest, making life easy ;-)
Using fixture make it much easier to e.g. control the scope of the function, setup and teardown variables, set a group of testing parameters, etc.  more info see https://docs.pytest.org/en/6.2.x/fixture.html#fixture-availability.